sdk/dataprotection/Azure.ResourceManager.DataProtectionBackup/src/Customized/Models/DataSourceInfo.cs

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#nullable disable
+
+using System;
+using System.ComponentModel;
+
+namespace Azure.ResourceManager.DataProtectionBackup.Models
+{
+    /// <summary> Datasource to be backed up. </summary>
```

Comment:
Let us remove this document (because the other partial class already has it)

Reply:
Fixed